agent-connection/src/main/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/PythonRefAgentConnection.kt

```diff
@@ -266,18 +264,10 @@ class PythonRefAgentConnection : AgentConnection {
         return Single.create { observer ->
```

Comment:
There is trivial logic and I don't see reasons to be as separated function. Could we insert it directly to callers?

Reply:
I think that such naming makes it transparent for others, about what we are doing here. And it is good practice to have same logic joint to functions)